custom/enikshay/private_sector_datamigration/factory.py

```diff
@@ -120,6 +126,12 @@ def get_person_case_structure(self):
 
         kwargs['attrs']['update']['facility_assigned_to'] = self._location_owner_id
 
+        if self.beneficiary.pincode:
+            kwargs['attrs']['update']['current_address_postal_code'] = self.beneficiary.pincode
+
+        if self.beneficiary.villageTownCity is not None:
+            kwargs['attrs']['update']['current_address_village_town_city'] = self.beneficiary.villageTownCity
```

Comment:
just wondering - why are these two only set if they're not null, when the other ones are all set unconditionally?